ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -240,3 +240,9 @@ oc exec -it  pod/mariadb-openstack -- mysql -uroot -p${MARIADB_PASSWD} -e "show
 # Post tests for keystone-operator
 # Check to confirm you can issue a token.
 openstack token issue
+
+# NOTE: if manila is deployed, build a default share required by tempest
+if [[ "${SERVICE_NAME}" == "MANILA" ]]; then
+    openstack share type create default false
```

Comment:
Testing this in https://github.com/openshift/release/pull/38755 and getting the following error: `openstack: 'share type create default false' is not an openstack command. See 'openstack --help`. Logs: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/38755/rehearse-38755-pull-ci-openstack-k8s-operators-manila-operator-main-manila-operator-build-deploy-tempest/1658011422828793856/artifacts/manila-operator-build-deploy-tempest/openstack-k8s-operators-deploy/build-log.txt

Reply:
ah, ok I suspect the openstackcli isn't able to discover manila. Let me try something different, we'll keep investigating this part.